src/app/app.routes.ts

```diff
@@ -208,6 +208,27 @@ export const APP_ROUTES: Routes = [
           }
         ]
       },
+      {
+        path: 'search-libraries',
+        children: [
+          {
+            path: '',
+            component: SearchResultsComponent,
+            data: {
+              title: 'APP.BROWSE.SEARCH.TITLE',
+              reuse: true
+            }
+          },
+          {
+            path: 'preview/:nodeId',
+            loadChildren:
+              'src/app/components/preview/preview.module#PreviewModule',
```

Comment:
Please make it relative to the app.module like other routes. Otherwise it breaks for certain scenarios.